<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecodeNMEADialog</class>
 <widget class="QDialog" name="DecodeNMEADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>$GPGLL</string>
     </property>
     <widget class="QLineEdit" name="le_gll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>531</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>$GPGSA</string>
     </property>
     <widget class="QLineEdit" name="le_gsa">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>$GPRMC</string>
     </property>
     <widget class="QLineEdit" name="le_rmc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>531</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>$GPGSV, $GLGSV</string>
     </property>
     <widget class="QPlainTextEdit" name="pte_gsv">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>571</width>
        <height>101</height>
       </rect>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>$GPVTG</string>
     </property>
     <widget class="QLineEdit" name="le_vtg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>$GPZDA</string>
     </property>
     <widget class="QLineEdit" name="le_zda">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>$GPGGA</string>
     </property>
     <widget class="QLineEdit" name="le_gga">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>$PMTKCHN</string>
     </property>
     <widget class="QLineEdit" name="le_chn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
